bugsnag-android-core/src/test/resources/event_serialization_0.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I find it quite difficult to link these json files back to the original test cases (and it's not obvious what the differences are without studying them)

Is there anyway we can name these files in a more meaningful way (even if that means having to specify the name against the test cases)?

Reply:
We have a method that allows the resource filename to be passed in as an argument when comparing the JSON generated by an object. We can use this with new test cases going forward.